src/layouts/partials/DrawerMainMenu/DrawerMainMenu.jsx

```diff
@@ -17,6 +17,7 @@ import {
   DUCK_PERSIST_ACCOUNT,
 } from '@chronobank/core/redux/persistAccount/constants'
 import { logoutAndNavigateToRoot } from 'redux/ui/thunks'
+import defaultAvatar from 'assets/img/profile-photo-1.jpg'
```

Comment:
Please use constant DEFAULT_AVATAR from `src/components/constants.js`
Or move this constant to another appropriate place and use it after that.